pubspec.yaml

```diff
@@ -88,3 +90,5 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+  assets:
+    - movieImages/
```

Comment:
use the file convention for folders, lower case separated with underscores

Reply:
fixed